docs/7___appendix.adoc

```diff
@@ -53,7 +53,7 @@ For details see <<MS93>>.
 |Something that occurs instantaneously at a specific time or when a specific condition occurs.
 At an event in Model Exchange, numerical integration is suspended and variables may change their values discontinuously.
 In Co-Simulation, an event may interrupt a <<fmi3DoStep>> to introduce an additional communication point.
-Internal events occur inside the FMU and should be signaled to the environment without any delay (in CS using <<early-return,early return>>, in SE using <<IntermediateUpdateMode,intermediate update>>) and can cause event handling and/or the activation of an <<outputClock,output Clock>>.
+Internal events occur inside the FMU and should be signaled to the environment without any delay (in CS using <<early-return,early return>>, in SE using <<ClockUpdateMode>>) and can cause event handling and/or the activation of an <<outputClock,output Clock>>.
```

Comment:
So if you say for countdown clocks the interval T_interval starts at t_event when getInterval is called then getInterval might have to be mentioned where as well. Not sure if that would help or confuse the reader.

Reply:
Event Mode describes and allowes fmi3GetInterval() - what am I missing? Maybe I do not get your point...

Reply:
The next tick of a countdown clock is defined as t_**event**  + T_interval. So there occurs an event before the countdown clock is ticking. Should that case be mentioned here in the def for event in the appendix?